commit 3f02b2be383bb2ab233fe8885f062f75fd902e78
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1332366465 -0500

    Change tqaddAction -> addAction. There is no tqaddAction defined in tqt headers.

diff --git a/kdiff3plugin/kdiff3plugin.cpp b/kdiff3plugin/kdiff3plugin.cpp
index ec008f1..02fa72d 100755
--- a/kdiff3plugin/kdiff3plugin.cpp
+++ b/kdiff3plugin/kdiff3plugin.cpp
@@ -150,7 +150,7 @@ KDiff3Plugin::KDiff3Plugin( KonqPopupMenu* pPopupmenu, const char *name, const T
    pActionMenu->insert (pAction);
 
    addSeparator();
-   tqaddAction( pActionMenu );
+   addAction( pActionMenu );
    addSeparator();
 }
 
